ui/locales/empty/assets-de-empty.json

```diff
@@ -0,0 +1,111 @@
+{
```

Comment:
@nOester was hat es mit diesen Files auf sich?

Reply:
@TIL-EBP die kommen aus dem Branch feature/viewer-1344-locale-keys-bereinigen. Die habe ich wahrscheinlich nicht richtig emerged.

Reply:
@nOester Okay vermutlich hast du dann diesen Branch hier auf Basis des `feature/viewer-1344-locale-keys-bereinigen` Branches erstellt? Gehören die Files hier schon rein? Ansonsten können wir das gerne noch gemeinsam anschauen, wie du die Änderungen wieder rausbekommst

Reply:
Hallo @TIL-EBP 
Die Files gehören in den `feature/viewer-1344-locale-keys-bereinigen`. Ja, wäre super, wenn wir das gemeinsam ansehen könnten. Ich schreib dir via Teams.

Reply:
@TIL-EBP Könntest Du dir noch mal alle SonarCloud Issues mit dem Tag `checkwithdev` anshen? Insbesondere, die mit dem Status `open`

Reply:
@nOester ich habe sie mir alle mal angeschaut. Diejenigen in den "swissforage" Files habe ich mal ignoriert, da diese Files ja bald entfernt werden sollen (simmt das noch?). Andere habe ich akzeptiert und bei zweien habe ich Anpassungen am code gemacht und hier gepusht.

Reply:
Deine Änderungen schaue ich mir gleich noch an. Dann kannst du von mir aus den PR auch aus dem Draft modus nehmen, wenn du durch bist